CourseGrab/Course.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Course.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 1/26/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+enum Status: String, Codable {
```

Comment:
I'm skeptical of making this have a String raw type. I'm not sure what the best alternative is (or if this is a real concern), but personally it looks a little off to me.

Reply:
also curious whats the reason we're making it a `String`?